components/Register/index.js

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable jsx-a11y/accessible-emoji */ // FIxed in emoji component
+import { useRouter } from "next/router";
+import {
+  Typography,
+  Form,
+  Input,
+  Button,
+  Select,
+  DatePicker,
+  Upload,
+  Space,
+  Row,
+  Col,
+} from "antd";
+import { UploadOutlined } from "@ant-design/icons";
+import { useAuth } from "~/components/Auth";
+import * as api from "~/lib/utils/api.js";
+import Emoji from "../Emoji";
+
+import styles from "./style.module.css";
+
+const { Option } = Select;
+
+const CountrySelect = () => (
+  <Select defaultValue="+351">
+    <Option value="+351">
+      <Emoji label="Bandeira Portuguesa">🇵🇹</Emoji>
+    </Option>
+  </Select>
+);
+
+function Register() {
+  const router = useRouter();
+  const { Title } = Typography;
+  const { user, errors, isLoading } = useAuth();
+
+  const onFinish = (e) => {
+    api
+      .registerMentor({
+        user_id: user.id,
+        first_name: e.first_name,
+        last_name: e.last_name,
+        mobile: e.mobile,
+        photo: e.photo?.file,
+        // birtday,
+        major: e.major,
+      })
+      .then(() => router.push("/dashboard"));
+  };
+
+  return (
+    <>
+      <Row
+        justify="center"
+        align="middle"
+        gutter={12}
+        className={styles.header}
+      >
+        <Col>
+          <Emoji label="Party Popper Emoji" className={styles.emoji}>
+            🎉
+          </Emoji>
+        </Col>
+        <Col>
+          <Title style={{ margin: 0 }}>Já está quase!</Title>
+          <Title level={3} style={{ margin: 0 }}>
+            Basta preencheres o formulário abaixo para terminares o registo
+          </Title>
+        </Col>
+      </Row>
+
+      {/* <Row justify="center">
+        <Avatar
```

Comment:
Will we ever have a Avatar on the register component? :thinking:

Reply:
Yeah, it is supposed to change to an image when the user uploads it, but I'm not sure how to do it right now because of how the upload works. As it's not a priority it will stay commented as a TODO